src/BootstrapMixin.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styleMaps from './styleMaps';
-import CustomPropTypes from './utils/CustomPropTypes';
+import { keyOf } from 'react-prop-types';
```

Comment:
I was thinking about how often we recommend to people to do this with react-bootstrap components so they don't bring in the entire library if they don't need it. If we continue to do that then we should make an effort to keep those components as lean as possible.